invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -0,0 +1,122 @@
+// This file is part of InvenioRDM
```

Comment:
It might be much more verbose, but IMO is much more readable and understandable :)